python/tk_multi_loader/ui/dialog.py

```diff
@@ -97,7 +99,7 @@ def setupUi(self, Dialog):
         self.top_toolbar.addWidget(self.label)
         self.left_area.addLayout(self.top_toolbar)
         self.entity_preset_tabs = QtGui.QTabWidget(Dialog)
-        self.entity_preset_tabs.setMaximumSize(QtCore.QSize(300, 16777202))
+        self.entity_preset_tabs.setMaximumHeight(16777202)
```

Comment:
Isn't there a way to use QSize to have it flex within the layout it's contained in?